testcases/contenttype.http

```diff
@@ -0,0 +1,11 @@
+POST / HTTP/1.1
+Host: handler.example.com
+Content-Type: application/xml
+Content-Length: 24
+ce-specversion: 0.3
```

Comment:
these needs to be `0.4-wip` to align with all of the other samples in our docs

Reply:
Done.